modules/pytket-braket/pytket/extensions/braket/backends/braket.py

```diff
@@ -183,25 +185,47 @@ class BraketBackend(Backend):
     def __init__(
         self,
         local: bool = False,
-        s3_bucket: str = "",
-        s3_folder: str = "",
-        device_type: str = "quantum-simulator",
-        provider: str = "amazon",
-        device: str = "sv1",
+        device: Optional[str] = None,
+        s3_bucket: Optional[str] = None,
+        s3_folder: Optional[str] = None,
+        device_type: Optional[str] = None,
+        provider: Optional[str] = None,
     ):
         """
         Construct a new braket backend.
 
         If `local=True`, other parameters are ignored.
 
+        All parameters except `device` can be set in config using
```

Comment:
Why not "device"?

Reply:
just to match the existing interfaces of other backends where the device is always specified at code-time. Willing to change this later